pipeline/src/org/labkey/pipeline/mule/EPipelineQueueImpl.java

```diff
@@ -307,9 +307,14 @@ public static void dispatchJob(PipelineJob job) throws UMOException
         MuleClient client = null;
         try
         {
+            _log.debug("Dispatching job: " + job);
             client = new MuleClient();
             client.dispatch(EPipelineQueueImpl.PIPELINE_QUEUE_NAME, job, null);
         }
+        catch (Exception e)
+        {
+            _log.debug("Error dispatching job: " + job, e);
+        }

```

Comment:
```suggestion
        catch (UMOException | RuntimeException e)
        {
            _log.debug("Error dispatching job: " + job, e);
            throw e;
        }
```

I haven't tested this (or even compiled it, actually) but we need to rethrow to avoid changing the execution path.

Reply:
good point

Reply:
@labkey-jeckels  Why limit to just two exception classes if we're just going to rethrow it anyway?